mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -278,6 +278,11 @@ SONIC:
 		Offset: -170,0,0
 	AutoTarget:
 	WithVoxelTurret:
+	Explodes:
+		RequiresCondition: !rank-elite
+	Explodes@ELITE:
+		RequiresCondition: rank-elite
+		Weapon: UnitExplode
```

Comment:
Could you please clarify for me why EmptyWeapon for !rank-elite has inherited value "UnitExplodeSmall", but EmptyWeapon for rank-elite has default value "UnitExplode"?
(so for tanks it is UnitExplodeSmall+UnitExplodeSmall, for Sonic UnitExplode+UnitExplode)